web/src/compose_actions.ts

```diff
@@ -331,6 +334,16 @@ export function start(raw_opts: ComposeActionsStartOpts): void {
         // display that it's too long.
         compose_validate.check_overflow_text();
     }
+    // This has to happen after we insert the content, so that the next "input" event
+    // is from user input.
+    if (restoring_last_draft) {
+        compose_state.set_restored_message_is_unedited(true);
+        $("textarea#compose-textarea")
+            .off("input")
```

Comment:
This seems dangerous -- isn't it possible this would destroy on-input event handlers on the textarea beyond just the one we're trying to add?

Reply:
Ah yeah sorry for not looking more into if there were other input events there (which there totally are). I was trying to find a way to not have performance issues form checking something on every input event.

Does this seem reasonable? It would just check a constant each time, which isn't too much work

```js
    $("textarea#compose-textarea").on("input", () => {
        if (compose_state.get_is_restored_draft_content_unedited()) {
            compose_state.set_is_restored_draft_content_unedited(false);
        }
    });
```

Reply:
I think the cleaner option would be to just add this to existing handler we have for `$("textarea#compose-textarea").on("input propertychange", () => {`